src/Sql/Sql/help/Get-AzSqlInstanceDtc.md

```diff
@@ -0,0 +1,188 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
```

Comment:
Please fill in the online version, using the same format as other docs.

Reply:
Filled